src/scores/probability/brier_impl.py

```diff
@@ -61,3 +67,155 @@ def brier_score(
         check_binary(obs, "obs")
 
     return mse(fcst, obs, reduce_dims=reduce_dims, preserve_dims=preserve_dims, weights=weights)
+
+
+def ensemble_brier_score(
+    fcst: XarrayLike,  # type: ignore
+    obs: XarrayLike,  # type: ignore
+    ensemble_member_dim: str,
+    thresholds: Union[float, int, Sequence[float]],
```

Comment:
ideally, since "thresholds" is ambiguous, it might be useful to indicate what kind of thresholds we're dealing with, e.g. event_thresholds.